qlib/config.py

```
@@ -76,7 +80,7 @@ def set_conf_from_C(self, config_c):
         self.update(**config_c.__dict__["_config"])
 
     @staticmethod
-    def register_from_C(config, skip_register=True):
+    def register_from_C(config, skip_register=False):  # Changed default parameter
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)